pages/vi/vi-vagrant.md

```diff
@@ -4,7 +4,7 @@
 
 ## Prerequisites
 
-Before installing Vagrant on any platform, it is necessary to check if VT-x/AMD-V instruction set is enabled on your processor by checking the BIOS. This is a requirement for installing vagrant on any platform since Vagrant is a type of virtualization software that utilizes VirtualBox. Most recent CPUs have this feature enabled already. If you are having trouble running vagrant, it may just be the case that VT-x/AMD-V is not enabled on your system.
+Before installing Vagrant on any platform, it is necessary to check if VT-x/AMD-V instruction set is enabled on your processor by checking the BIOS. This is a requirement for installing vagrant on any platform since Vagrant is a type of virtualization software that utilizes VirtualBox. Most recent CPUs have this feature enabled already. Another requirement is to have Hyper-V enabled in your Windows Features, which allows Windows to create virtual machine's on your computer. If you are having trouble running vagrant, it may just be the case that VT-x/AMD-V and Hyper-V is not enabled on your system.  
```

Comment:
Hello NickBratt, I noticed there are some minor errors here.

First problem: "[...] This is a requirement for installing vagrant on [...]". The word "vagrant" here need to be uppercase.

Second problem: In my opinion, can you change the sentence "Another requirement is to have ...." into a note block at the end of the paragraph. For example, "Note: For Windows user, another requirement is to have...". Since our paragraph is currently talking about all operating system as a whole. If we have this sentence included then it will not make sense.

Final problem: I think you can add this [link](https://blogs.technet.microsoft.com/canitpro/2015/09/08/step-by-step-enabling-hyper-v-for-use-on-windows-10/) to help the user know how to enable Hyper-V on Windows 10.

Other than those problems, everything else is pretty solid. What do other vi think?